src/app/Dashboard/AddCard.tsx

```diff
@@ -240,37 +242,37 @@ export const AddCard: React.FunctionComponent<AddCardProps> = (props: AddCardPro
 interface PropsConfigFormProps {
   cardTitle: string;
   controls: PropControl[];
-  config: any;
-  onChange: ({}) => void;
+  config: object;
+  onChange: (config: object) => void;
 }
 
-const PropsConfigForm = (props: PropsConfigFormProps) => {
+const PropsConfigForm = ({ onChange, ...props }: PropsConfigFormProps) => {
```

Comment:
Can we destructure `config` from props here too?

Reply:
See discussion: https://github.com/cryostatio/cryostat-web/pull/780#discussion_r1054640619